client/src/components/ReviewsList.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import ReviewsListEntity from './ReviewsListEntity.jsx';
+import ReviewsPages from './ReviewsPages.jsx';
+
+class ReviewsList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+        ordered_reviews: this.props.reviews,
+        rendering_order: 'Most recent',
+        page_num: 1
+    }
+
+    this.changeReviewsOrder = this.changeReviewsOrder.bind(this);
+    this.changeReviewsPage = this.changeReviewsPage.bind(this);
+  }
+
+  changeReviewsOrder(event) {
+      // sort reviews according to selected order
+      let order = event.target.value;
+      let ordered_reviews = [];
+
+      if (order === 'Most recent') {
+        ordered_reviews = this.props.reviews.sort( (a,b) => {
+            return new Date(b.update_date) - new Date(a.update_date);
+        });
+
+      } else if (order === 'Highest rated') {
+        ordered_reviews = this.props.reviews.sort( (a,b) => {
+            return b.rating - a.rating;
+        });
+
+      }
+
+      this.setState({
+          ordered_reviews: ordered_reviews,
+          rendering_order: order,
+          page_num: 1
+      });
+  }
+
+  changeReviewsPage(event) {
+
+      let value = parseInt(event.target.className);
```

Comment:
It's best practice to always specify the radix when using parseInt (i.e. you should put 10 as the second argument). It's OK in this case, but there are edge cases where you get bugs if you don't. See the link: https://stackoverflow.com/questions/7318385/problems-with-javascript-parseint